README.rst

```diff
@@ -346,72 +346,74 @@ Build and Run Application
 
 The DiskPerformanceApp can be built and packaged using the Apache Maven command::
 
-    mvn clean package
+  $ mvn clean package
 
 Note that the remaining commands assume that the ``cdap-cli.sh`` script is available on your PATH.
 If this is not the case, please add it::
 
-    export PATH=$PATH:<CDAP home>/bin
+  $ export PATH=$PATH:<CDAP home>/bin
 
 If you haven't already started a standalone CDAP installation, start it with the command::
 
-    cdap.sh start
+  $ cdap.sh start
 
 We can then deploy the application::
 
-    cdap-cli.sh deploy app target/cdap-flow-guide-1.0.0.jar
+  $ cdap-cli.sh deploy app target/cdap-flow-guide-<version>.jar
 
 Next we start the flow::
 
-    cdap-cli.sh start flow DiskPerformanceApp.DiskPerformanceFlow
+  $ cdap-cli.sh start flow DiskPerformanceApp.DiskPerformanceFlow
 
 Note that there is one instance of the ``Detector`` Flowlet running and two instances of the ``Tracker`` Flowlet running::
 
-    cdap-cli.sh get instances flowlet DiskPerformanceApp.DiskPerformanceFlow.slowReadDetector
-    1
+  $ cdap-cli.sh get flowlet instances DiskPerformanceApp.DiskPerformanceFlow.slowReadDetector
+  1
 
-    cdap-cli.sh get instances flowlet DiskPerformanceApp.DiskPerformanceFlow.slowDiskTracker
-    2
+  $ cdap-cli.sh get flowlet instances DiskPerformanceApp.DiskPerformanceFlow.slowDiskTracker
+  2
 
 We can scale out our application and increase the number of ``Tracker`` Flowlets to four::
 
-    cdap-cli.sh set instances flowlet DiskPerformanceApp.DiskPerformanceFlow.slowDiskTracker 4
-
-    cdap-cli.sh get instances flowlet DiskPerformanceApp.DiskPerformanceFlow.slowDiskTracker
-    4
+  $ cdap-cli.sh set flowlet instances DiskPerformanceApp.DiskPerformanceFlow.slowDiskTracker 4
+  Successfully set flowlet 'DiskPerformanceFlow' of flow 'slowDiskTracker' of app 'DiskPerformanceApp' to 4 instances
+  
+  $ cdap-cli.sh get flowlet instances DiskPerformanceApp.DiskPerformanceFlow.slowDiskTracker
+  4
 
 Scaling your application is easy in CDAP!
 Now we can manually send enough slow disk events to the diskReads stream to get a disk classified as a slow disk::
 
-    cdap-cli.sh send stream diskReads "disk1 1001"
-    cdap-cli.sh send stream diskReads "disk1 1001"
-    cdap-cli.sh send stream diskReads "disk1 1001"
-    cdap-cli.sh send stream diskReads "disk1 1001"
+  $ cdap-cli.sh send stream diskReads \''disk1 1001'\'
```

Comment:
I have tested this. The problem is the cdap-cli.sh; see https://issues.cask.co/browse/CDAP-2023. This is calling the CLI from the command line, which (unfortunately) is different than the same command from inside the CDAP shell.